### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by undergroundwires** [amsik/liquor-tree#244]
First of all thank you a lot for this component. Well-documented, easy-to-use and zero dependencies. I use this component as main UI in [privacy.sexy](https://github.com/undergroundwires/privacy.sexy).

However latest commit is two years ago and we do not get attention for issues/PRs. 

It would be nice to get some official information @amsik .  How do you see the future of this project? Did you dump it completely, or do you plan to maintain it again?

**Comment by dogukanoksuz** [comment]
@undergroundwires Did you find any alternative components that doesn't require too much effort to migrate?

**Comment by undergroundwires** [comment]
@dogukanoksuz Maalesef hocam, I'm in process of rewriting this component as it blocks migration to Vue 3.0. Lessons learnt so I will not go for any third-party tree component but maintain a simple one myself.

**Comment by timz** [comment]
> @dogukanoksuz Maalesef hocam, I'm in process of rewriting this component as it blocks migration to Vue 3.0. Lessons learnt so I will not go for any third-party tree component but maintain a simple one myself.

did you succeed? Would you like to share?

**Comment by undergroundwires** [comment]
Yes, running it on production. I have rewritten it from scratch. It uses typescript, Vue 3, hooks etc with completely different internals, API and less functionality. You can [see the source code here](https://github.com/undergroundwires/privacy.sexy/tree/0.12.10/src/presentation/components/Scripts/View/Tree/TreeView) and [this is how it's used](https://github.com/undergroundwires/privacy.sexy/blob/6ada8d425c4a7df88490756187c84b5c57ed1dcc/src/presentation/components/Scripts/View/Tree/ScriptsTree.vue#L8-L18).

**Comment by curiosus42** [comment]
@undergroundwires Nice work with the port!
Would you consider forking this project with your Vue 3 version of it?
